pallets/sidechain/src/lib.rs

```diff
@@ -95,4 +103,26 @@ pub mod pallet {
 			}
 		}
 	}
+
+	/// Priviledged extrinsic to atomically upgrade runtime code and vital sidechain parameters
```

Comment:
Please add params docs for
* `genesis_utxo` - that is should be an UTXO already burned to `init-governance` with v7 (it is not 00...00#0)
* `selection_main_chain_scripts` addresses and policies obtained for with v7 for the `genesis_utxo` above
* `code` please remind users that they have to update `spec_version` of Runtime code.

Also please update spec_version, I suggest 131 in our code.